views/BD103.html

```diff
@@ -4,13 +4,39 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<meta http-equiv="refresh" content="3; URL=https://bd103.github.io">
-	<title>BD103 Woo</title>
+	<title>BD103</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Pangolin&display=swap');
+    body {
+      font-family: "Montserrat", sans-serif;
+      font-size: 20px;
+      background: linear-gradient(to bottom right, darkcyan, lightcyan);
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-attachment: fixed;
+    }
+    h1, h2 {font-family: "Pangolin", cursive;}
+    .title {font-size: 60px;}
+    .center {text-align: center;}
+    .left {text-align: left;}
+    .right {text-align: right;}
+  </style>
```

Comment:
Umm sorry, I also have a problem with this. I already said that you should put your CSS in the style.css file(I even deleted everyone's files earlier cause they didn't do that). It is tidier that way. What I told everyone is to put a class on their HTML tag, ex. `<HTML class="bd">`, and then in the style.css file they can create a section for themselves, with comments, and for the CSS properties we could have:
```html
.bd body {
      font-family: "Montserrat", sans-serif;
      font-size: 20px;
      background: linear-gradient(to bottom right, darkcyan, lightcyan);
      background-repeat: no-repeat;
      background-size: cover;
      background-attachment: fixed;
 }
.bd h1, h2 {font-family: "Pangolin", cursive;}
.bd .title {font-size: 60px;}
.bd .center {text-align: center;}
.bd .left {text-align: left;}
.bd .right {text-align: right;}
```
And you could add the @import font thing at the top of the file with the other ones.